### DISCUSSION TITLE
TFT35 E3 v3.0.1 (GD Version) Firmware Issue Resolved

### DISCUSSION THREAD

**Discussion by lhardy5150** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2887] [General]
I have the GD version TFT35-E3 v3.0.1. I selected 'BIGTREE_GD_TFT35_V3.0_E3.27.x.bin file', the 'config.ini' and the 'TFT35' folder within the 'THEME_Unified Menu Material theme' folder from the '[Copy to SD Card root directory to update]'. 

I insert card into board, powered up the unit using USB 5V and no printer cables connected.  It loads into the firmware updating screen, loads to 100% then it goes to a black screen.  I tried several SD cards thinking that perhaps that was causing the issue and it made no difference. I tried the reset.txt file and that didn't work. It felt like it wasn't the correct fw for v3.0.1. 

Well. After two days wasted trying to figure this out and almost buying another TFT, turns out the BIGTREE_GD_TFT35_V3.0_E3.27.x.bin file is messed up. I happened to find a firmware bin file V1 Engineering did for their CNC product and they were using the exact GD version of the Board. https://github.com/V1EngineeringInc/BIGTREETECH-TouchScreenFirmware/releases Downloaded CNC_TFT.zip and placed the files on the card and it worked!! 

I compared the config.ini file with the one on the official github site and there wasn't anything extremely different. I decided to replace the working files with the ones on the official github site, one by one, to see which one was causing the issue. I tried the config.ini file as well as the TFT folder within the THEME_Unified Menu Material theme folder and they loaded fine with the V1 Engineering bin file. Changed to the official bin file and it failed. So there u go. Switched it back to V1 Engr bin file and it worked. 

That BIGTREE_GD_TFT35_V3.0_E3.27.x.bin file located on the official github site needs to be fixed.

I have attached the files that worked for my unit.  You may need to tweak the config file to match your printer config.

[BIGTREE GD TFT35 V3.0.1 FIRMWARE.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/14002728/BIGTREE.GD.TFT35.V3.0.1.FIRMWARE.zip)

**Comment by kisslorand** [comment]
You can also try the FW from [my repository](https://github.com/kisslorand/BTT-TFT-FW).
